### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by alexdukeb**

### CODE REVIEW

File: src/views/ProfileView.twig [hunk 1]
```diff
@@ -24,38 +24,37 @@
...
-                    <div class="button button--green">Reprendre</div>
-                </div>
-            </a>
+        <h2 class="title">Mes séries exercices {{ user.type == "student" ? "réalisées" : "créées" }}</h2>
+
+        <div class="profile__series">
+            {% for serie in series %}
+                <a href="" class="exercise">
+                    <h4>{{serie.title}}</h4>
+                    <p>{{exo.author}}</p>
```
Comment (AnthonyZink) [inline src/views/ProfileView.twig]: Copy and paste isn't a good idea 😄 
exo should be serie

File: models/Exercise.js [hunk 2]
```diff
@@ -38,6 +38,11 @@ ExerciseSchema.statics.getExo = function (data, callback) {
     Exercise.findOne(query, callback);
 };
 
+ExerciseSchema.statics.ByAuthor = function (author, callback) {
+    let query = { author: author };
```
Comment (AnthonyZink) [inline models/Exercise.js]: Es6 bro : {author: author} should be {author}

File: models/Set.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        default: []
+    },
+    author: {
+        type: String,
+        required: true
+    }
+});
+
+SetSchema.statics.ByAuthor = function (author, callback) {
+    let query = { author: author };
```
Comment (AnthonyZink) [inline models/Set.js]: Es6 too

Comment (alexdukeb) [reply models/Set.js]: Fixed

File: models/Exercise.js (hunk 2 above)
Comment (alexdukeb) [reply models/Exercise.js]: Fixed

File: src/views/ProfileView.twig (hunk 1 above)
Comment (alexdukeb) [reply src/views/ProfileView.twig]: Fixed

**Pull request merged**